src/display.rs

```diff
@@ -11,16 +11,43 @@
 
 use libc;
 use base::CGError;
+use geometry::{CGRect, CGPoint, CGSize};
 
 pub type CGDirectDisplayID = libc::uint32_t;
 
 #[link(name = "ApplicationServices", kind = "framework")]
 extern {
     pub fn CGMainDisplayID() -> CGDirectDisplayID;
+    pub fn CGDisplayIsActive(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsAlwaysInMirrorSet(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsAsleep(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsBuiltin(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsInHWMirrorSet(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsInMirrorSet(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsMain(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsOnline(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsStereo(display: CGDirectDisplayID) -> bool;
```

Comment:
No idea, I at first thought it might be part of libc but it's defined in `<mach/boolean.h>` which is in Kernel.framework which is a bunch of headers for interoperating with the kernel... doesn't sound like something this should really have a dependency on. For now defining it here might be your best bet, and we can refactor later if it starts to be used in more places.